R/fcast.R

```diff
@@ -251,5 +251,5 @@ dcast.data.table = function(data, formula, fun.aggregate = NULL, sep = "_", ...,
     setattr(ans, 'names', c(lhsnames, allcols))
     setDT(ans); setattr(ans, 'sorted', lhsnames)
   } else stopf("Internal error -- empty rhsnames in dcast; please report") # nocov
-  return (ans)
+  return(ans)
```

Comment:
is there a linter to avoid return at end of function? (I only recommend using return for exiting early in R code)

Reply:
yes, return_lintee() 😄 

currently deactivated